CHANGELOG.md

```diff
@@ -5,11 +5,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [v2.16.0]
```

Comment:
Please add it to main as well

Reply:
please see https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/2522